source/layouts/layout.erb

```diff
@@ -16,11 +16,7 @@ under the License.
 <% language_tabs = current_page.data.language_tabs || [] %>
 <% page_content = yield %>
 <%
-  if current_page.data.includes
-    current_page.data.includes.each do |include|
-      page_content += partial("includes/#{include}")
-    end
-  end
+  page_content += partial("includes/_graphql.md.erb")
```

Comment:
we don't have any other includes right now, so i've just hardcoded the graphql. We can change this in the future if we want to pull out other things.